src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -49,10 +54,16 @@
 import com.amazon.opendistroforelasticsearch.security.securityconf.DynamicConfigModel;
 import com.amazon.opendistroforelasticsearch.security.user.User;
 
+import com.google.common.collect.ImmutableMap;
+
 public class PrivilegesInterceptorImpl extends PrivilegesInterceptor {
 
     private static final String USER_TENANT = "__user__";
     private static final String EMPTY_STRING = "";
+    private static final Map<String, Object> KIBANA_INDEX_SETTINGS = ImmutableMap.of(
+            IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1,
+            IndexMetadata.SETTING_AUTO_EXPAND_REPLICAS, "0-all"
```

Comment:
Is there need to set this as auto expand? This index is created for every user and may flood with replicas based on the number of nodes and users in the system.

Reply:
Likely not required, I copied settings from `.kibana` index.